src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for setting up cookie authentication services in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class CookieAuthenticationServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds cookie authentication services to the specified <see cref="IServiceCollection" />. 
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
+        public static IServiceCollection AddCookieAuthentication(this IServiceCollection services)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddAuthentication();
+            services.AddAntiforgery();
+            return services;
+        }
+
+        /// <summary>
+        /// Adds cookie authentication services to the specified <see cref="IServiceCollection" />. 
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configureOptions">An action delegate to configure the provided <see cref="SharedAuthenticationOptions"/>.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
+        public static IServiceCollection AddCookieAuthentication(this IServiceCollection services, Action<SharedAuthenticationOptions> configureOptions)
```

Comment:
This extension is horrible. TBH, I can't figure out why it would be so bad to simply reference the antiforgery package from the core authentication package as it's a rather light package and needs few dependencies. 

If the antiforgery block was a bit more flexible about the cookie name, we could even use it in RemoteAuthenticationHandler.